app/static/src/app/components/run/RunStochasticPlot.vue

```diff
@@ -43,7 +44,15 @@ export default defineComponent({
             if (!result) {
                 return [];
             }
-            return discreteSeriesSetToPlotly(filterSeriesSet(result, selectedVariables.value), palette.value);
+            const replicates = store.state.run.numberOfReplicates;
+            const maxReplicatesDisplay = (store.state.config as StochasticConfig)?.maxReplicatesDisplay || 50;
```

Comment:
This `|| 50` turns up in a couple of places - maybe we should just set it in the config by default if not provided - same with the 1000 for run max.

Reply:
Actually just did some digging, these are set in config by default: https://github.com/mrc-ide/wodin/blob/9d4d572139b368d241e3b064e13ce4ce35dd3795/app/server/src/controllers/configController.ts#L18, so i have removed the defaults

Reply:
can you increase this default to 50 please? 20 feels like not enough. I think 20 might have been inherited from the shiny version which got very slow due to moving data around but 50 here feels snappy in plotly and is somewhat interpretable. 100 feels like definitely too many.

Reply:
done, yh think 50 should be alright!